tasks/configure.yml

```diff
@@ -18,7 +18,7 @@
     mode: "0600"
   loop: "{{ barman_pg_servers }}"
   register: barman_server_configs_result
-  no_log: true
+  no_log: "{{ barman_no_log }}"
```

Comment:
`barman_no_log` -> `barman_no_log | bool` so that overriding from command line works in an obvious way (i.e. `-e barman_no_log=false`)